pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `$id`                       | Core          | 06                 | No      | Replaced `id`                                                                                |
+| `$defs`                     | Core          | 2019-09            | No      | Replaced `definitions`                                                                       |
+| `$dynamicAnchor`            | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`                                                                  |
+| `$dynamicRef`               | Core          | 2020-12            | No      | Replaced `$recursiveRef`                                                                     |
+| `$recursiveAnchor`          | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicAnchor`                                                                 |
+| `$recursiveRef`             | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicRef`                                                                    |
+| `$ref` (legacy)             | Core          | 03                 | 2019-09 | Replaced by `$ref`                                                                           |
+| `$ref`                      | Core          | 2019-09            | No      | Replaced `$ref` (legacy)                                                                     |
+| `$schema`                   | Core          | 03                 | No      | No                                                                                           |
+| `$vocabulary`               | Core          | 2019-09            | No      | No                                                                                           |
+| `additionalItems`           | Core          | 03                 | 2020-12 | Replaced by `items`                                                                          |
+| `allOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `anyOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `const`                     | Validation    | 06                 | No      | No                                                                                           |
+| `contains`                  | Core          | 06                 | No      | No                                                                                           |
+| `contentEncoding` (legacy)  | Core          | 01                 | 04      | Replaced by `media`.`binaryEncoding`                                                         |
+| `media`                     | Hyper-Schema  | 04                 | 07      | Replaced `contentEncoding` (legacy) and replaced by `contentEncoding` and `contentMediaType` |
```

Comment:
The keywords in this table are order alphabetically, so `media` here doesn't make sense?

Reply:
I only suggested including `media` in order to tell the complicated story of `contentEncoding`. In the beginning, there was only one spec for core, validation, and hyper-schema. I think it would make sense if we considered the original `contentEncoding` part of hyper-schema even though it wasn't a separate spec yet. Then we can remove these two entries and pretend it was introduced in draft-07 even though it's a little more complicated than that.